saml2idp/base.py

```diff
@@ -103,6 +110,8 @@ def _decode_request(self):
         """
         self._request_xml = base64.b64decode(self._saml_request)
 
+        self._logger.info('SAML request decoded', decoded_request=self._request_xml)
```

Comment:
I'm kinda on the fence whether this should log at `info` or `debug`... I think the XML payloads can be quite big and they're probably not something you'd always want to look at. I'd probably move it to `debug` for that reason.